libs/core/src/lib/utils/consts/error-messages.ts

```diff
@@ -1 +1,3 @@
 export const MOBILE_CONFIG_ERROR = `Missing configuration object for mobile version of the component. Consult the documentation on how to provide MobileModeConfiguration object for this component.`;
+
+export const THEME_SWITCHER_ROUTER_MISSING_ERROR = `You need to import RouterModule, to enable setting theme via url.`
```

Comment:
Discussed with Jessie and it should be `You need to import RouterModule to enable theme settings via the url.`